src/components/rangeUsePlanPage/versionsList/VersionsDropDown.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
+import useSWR from 'swr'
+import * as API from '../../../constants/api'
+import { axios, getAuthHeaderConfig } from '../../../utils'
+import VersionsDropDownList from './VersionsDropDownList'
+
+const sortVersions = (a, b) => {
+  if (b.version === -1) return 1
+  if (b.version > a.version) return 1
+  if (b.version < a.version) return -1
+  return 0
+}
+
+const VersionsDropDown = ({ match, open }) => {
+  const { planId } = match.params
+  const [selectedVersion, setSelectedVersion] = useState(null)
+  const endpoint = API.GET_RUP_VERSIONS(planId)
+
+  const { data, error, isValidating } = useSWR(endpoint, key =>
+    axios.get(key, getAuthHeaderConfig()).then(res => res.data)
+  )
+
+  const { versions = [] } = data || {}
+  const formattedVersions = versions
+    .sort(sortVersions)
+    .filter(v => v.version !== -1)
```

Comment:
We don't need this filter anymore. The `-1` was from a previous implementation where the current version was tracked by `-1`.

Reply:
Sounds good, 1b4bd3d